ntropy_sdk/ntropy_sdk.py

```diff
@@ -474,10 +475,9 @@ def add_transactions(
             raise KeyError(f"Missing columns {missing_cols}")
         overlapping_cols = set(mapping.values()).intersection(cols)
         if overlapping_cols:
-            raise KeyError(
-                f"Overlapping columns {overlapping_cols} will be overwritten"
-                "- consider overriding the mapping keyword argument, or move the existing columns to another column"
-            )
+            msg = f"Overlapping columns {overlapping_cols} will be overwritten"
+            msg += " - consider overriding the mapping keyword argument, or move the existing columns to another column."
+            warnings.warn(msg)
 
```

Comment:
We can use in place=False now (the new default) think that makes this PR superflupus